guides/TrainingModelsHPC.md

```diff
@@ -0,0 +1,169 @@
+# Train a detector in the cluster
+
+0.  **Prep**
+
+    - If not in the SWC network: connect to the SWC VPN.
+
+1.  **Connect to the SWC HPC cluster**
+
+    ```
+    ssh hpc-gw1
+    ```
+
+    It may ask for your password twice. To set up SSH keys for the SWC cluster, see [this guide](https://howto.neuroinformatics.dev/programming/SSH-SWC-cluster.html#ssh-keys).
+
+1.  **Fetch the training script from the 🦀 repository**
+
+    To do so, run any of the following commands. They will place the bash script for training in the current working directory. The files retrieved will be the versions in the `main` branch of the repository.
```

Comment:
Do we need to fetch? If they have the latest version of the repo, then it is technically there. Or is it to ensure the changes in the bash script being replace by fetching the file?

Reply:
Note that is not `git fetch`, but just downloading the file with `curl` - will change it to "download" for clarity.

It is easier than git cloning the repo to the login node, since we only need the bash scripts. The actual 🦀 package is installed in the compute node when running the bash script.